src/test_fixture/world.rs

```diff
@@ -99,56 +76,59 @@ impl TestWorldConfig {
     }
 }
 
+impl Default for TestWorld {
+    fn default() -> Self {
+        Self::new_with(TestWorldConfig::default())
+    }
+}
+
 impl TestWorld {
     /// Creates a new `TestWorld` instance using the provided `config`.
     /// # Panics
     /// Never.
-    pub async fn new_with(config: TestWorldConfig) -> TestWorld {
+    #[must_use]
+    pub fn new_with(config: TestWorldConfig) -> TestWorld {
```

Comment:
Good question - I changed it because new Infra does not need to block anymore: all the magic happens inside the transport layer. I could change it back as I certainly cannot guarantee that we will never need to make it async again, but I also don't mind doing it - keeping things async where they don't need to be is a bit weird.